src/components/person.tsx

```diff
@@ -0,0 +1,91 @@
+import classNames from 'classnames';
+import { useContext } from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import { MyContext } from './state';
+import { Person as PersonType } from '../types';
+
+interface PersonComponentProps {
+  pers: PersonType;
+}
+
+export const Person: React.FC<PersonComponentProps> = ({ pers }) => {
+  const { name, sex, born, fatherName, motherName, slug, died } = pers;
+  const { setSelectedPerson, person } = useContext(MyContext);
+  const location = useLocation();
+
+  const motherPerson = person.find(
+    (perso: PersonType) => perso.name === motherName,
+  );
+  const fatherPerson = person.find(
+    (perso: PersonType) => perso.name === fatherName,
+  );
+
+  const isPersonInUrl = location.pathname.includes(slug);
+
+  const getPeopleClass = () =>
+    classNames({ 'has-background-warning': isPersonInUrl });
+
+  return (
+    <tr data-cy="person" key={name} className={getPeopleClass()}>
+      <td>
+        <Link
+          to={`/people/${slug}`}
+          className={classNames({
+            'has-text-danger': sex === 'f',
+            'has-background-warning': isPersonInUrl,
+          })}
+          onClick={() => setSelectedPerson(slug)}
+        >
+          {name}
+        </Link>
```

Comment:
Code quality issue: You are using `key` attribute on a `tr` element. The `key` attribute should be used on the list of `tr` elements, not on the `tr` itself. This is because React uses the `key` attribute to determine which items in a list have changed, are added, or are removed.